.github/workflows/testing.yml

```
@@ -0,0 +1,46 @@
+        - name: Test with unittest
+          run: |
+            coverage run -m unittest discover
+
+        - name: Create Coverage Report
+          run: |
+            coverage report -m
+            coverage xml
+
+        - name: DeepSource Test Coverage
+          uses: deepsourcelabs/test-coverage-action@master
+          with: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)